benchmark/run_from_branch.sh

```diff
@@ -10,14 +10,19 @@ cd $TMP_DIR
 echo "Installing the dependencies."
 pip install pipenv
 # this is to install benchmark dependencies
+pipenv --rm
 pipenv install --dev --skip-lock
 # this is to install the AEA in the Pipenv virtual env
-pipenv run pip install --upgrade aea[all]=="0.11.1"
+pipenv run pip install --upgrade .[all] -y
+pipenv run pip uninstall typing -y
+pipenv run pip install aea-ledger-fetchai -y
+pipenv run pip install aea-ledger-cosmos -y
+pipenv run pip install aea-ledger-ethereum -y
```

Comment:
`--no-deps` missing, o/w it will install `aea` from PyPI

Reply:
done